flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -1023,25 +1014,63 @@ protected AbstractStateBackend createInnerBackend(Configuration config) {
 	// ------------------------------------------------------------------------
 	// ------------------------------------------------------------------------
 
+	private enum Event {
+		TASK_IS_RUNNING,
+	}
+
 	private static class EmptyStreamTask extends StreamTask<String, AbstractStreamOperator<String>> {
```

Comment:
I think this is no longer an "empty" `StreamTask` so it should be renamed. Maybe to `AwaitableStreamTask`? `MockStreamTask`? `TestingStreamTask`?

Reply:
My interpretation, that this task was called "empty" due to not emitting/producing anything.

But I haven't considered renaming it, to keep changes minimum. I can rename it if that not an issue.

Reply:
I'm not sure. I think that `EmptyStreamTask` is not the best, but also I don't think that my proposal are much better. I would lean towards `AwaitableStreamTask` or `AwaitableEmptyStreamTask`, but if you don't like it, I'm fine with leaving it as it is.

Reply:
I have renamed to `MockStreamTask`.